test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlAttributeTest.cs

```diff
@@ -329,12 +362,12 @@ public void ConditionalAttributesDoNotCreateExtraDataForEntirelyLiteralAttribute
         [Fact]
         public void ConditionalAttributesAreDisabledForDataAttributesInBlock()
         {
-            ParseBlockTest("<span data-foo='@foo'></span>",
+            ParseBlockTest("<span data-foo  =  '@foo'></span>",
```

Comment:
`data-` attributes are special and the only tests covering them in this class are this method and the next one. Restore the previous cases and _add_ these new cases.